packages/react-native/React/Views/ScrollView/RCTScrollView.m

```diff
@@ -970,7 +970,7 @@ - (void)uiManagerWillPerformMounting:(RCTUIManager *)manager
             CGPointMake(self->_scrollView.contentOffset.x, self->_scrollView.contentOffset.y + deltaY);
         if (autoscrollThreshold != nil) {
           // If the offset WAS within the threshold of the start, animate to the start.
-          if (y - deltaY <= [autoscrollThreshold integerValue]) {
+          if (y <= [autoscrollThreshold integerValue]) {
```

Comment:
To ease the review, the Fabric equivalent code is [here](https://github.com/facebook/react-native/blob/6e42c98aa3423cfbc389e4fe877b5c6b559c8a81/packages/react-native/React/Fabric/Mounting/ComponentViews/ScrollView/RCTScrollViewComponentView.mm#L781)